### DISCUSSION TITLE
Sourcemaps generated for `npm run build` despite being turned off

### DISCUSSION THREAD

**Discussion by iKonrad** [vercel/next.js#30140] [Help]
Based on the documentation, sourcemaps should not be generated by default for production. But for some reason they are generated when I run `npm run build`. I tried explicitly turn then off in the config as well, but it doesn't seem to work.

Am I missing something?

```
const withOptimizedImages = require('next-optimized-images');
const withBundleAnalyzer = require('@next/bundle-analyzer')({
  enabled: process.env.ANALYZE === 'true',
});
const { withSentryConfig } = require('@sentry/nextjs');

const SentryWebpackPluginOptions = {
...
};

module.exports = withSentryConfig(
  withBundleAnalyzer(
    withOptimizedImages({
      handleImages: ['jpeg', 'png', 'svg'],
      optimizeImagesInDev: true,
      trailingSlash: false,
      productionBrowserSourceMaps: false,
      images: {
        disableStaticImages: true,
      },
      typescript: {
        ignoreBuildErrors: false,
      },
    }),
  ),
  SentryWebpackPluginOptions,
);

```

![image](https://user-images.githubusercontent.com/3729309/138291254-139355f2-0f15-4d08-bb7c-23d69f7c0113.png)

**Comment by gaston-flores** [comment]
I can't reproduce this in a vanilla install of Next, but I think those are being generated by the Sentry plugin. [Sentry will generate sourcemaps](https://docs.sentry.io/platforms/javascript/guides/nextjs/manual-setup/#configure-source-maps) and upload them to their service as part of the production build.
